modules/crowdx/src/lib/store.ts

```diff
@@ -0,0 +1,159 @@
+import { Computed, IObservable, Kind } from './index'
+import Core from '../core'
+
+const reservedKeywords = [
+  '__crowdx_values__',
+  '__crowdx_deriveds__',
+  '__crowdx_observed__',
+  'constructor',
```

Comment:
Why is constructor reserved?

Reply:
Monkey brain sees method on class, monkey brain adds name to list of reserved keywords

Reply:
Upon further inspection, the constructor keyword is indeed a reserved keyword, as the 'constructor' property is always defined

Reply:
In that case shouldn't all of these be added? https://i.imgur.com/p8gmt2N.png

Reply:
Yup, they should. It has now been fixed